HW0/src/Main.java

```diff
@@ -2,4 +2,5 @@ public class Main {
     public static void main(String[] args) {
         System.out.print("Hello World");
     }
-}
\ No newline at end of file
+}
+//
```

Comment:
No significant meaning behind empty comment in the code. Just testing out commits before working on HW0